hq/markdown/run-command.md

```diff
@@ -92,14 +122,14 @@ for this_instance in $instance; do
    while [[ $responseCode -eq -1 ]]; do
       printf "."
       sleep 1
-      responseCode=$(aws --region $region --profile $profile ssm get-command-invocation \
+      responseCode=$(aws --region $region --profile $profile ssm list-command-invocations \
```

Comment:
This is a significant change!

cc @aug24 for approval

Reply:
Let's wait for @aug24 input, but the proposed version is correct. (What I think happened, but this is only a theory, is that @aug24 might have looked that the Windows PowerShell version, but @katebeeis correctly reporting the `aws` cli version.)

The AWS documentstion for `aws` says 

```
aws ssm list-command-invocations --command-id "command ID" --details
``` 

and @katebee proposes

```
aws ssm list-command-invocations \
    --region $region --profile $profile \
    --command-id "$command_id" \
    --instance-id "$this_instance" \
         | jq -r '.ResponseCode'
```

@adamnfish